fdbserver/storageserver.actor.cpp

```diff
@@ -3992,11 +3992,13 @@ bool rangeIntersectsAnyTenant(TenantPrefixIndex& prefixIndex, KeyRangeRef range,
 }
 
 TEST_CASE("/fdbserver/storageserver/rangeIntersectsAnyTenant") {
-	std::map<TenantName, TenantMapEntry> entries = { std::make_pair("tenant0"_sr, TenantMapEntry(0)),
-		                                             std::make_pair("tenant2"_sr, TenantMapEntry(2)),
-		                                             std::make_pair("tenant3"_sr, TenantMapEntry(3)),
-		                                             std::make_pair("tenant4"_sr, TenantMapEntry(4)),
-		                                             std::make_pair("tenant6"_sr, TenantMapEntry(6)) };
+	std::map<TenantName, TenantMapEntry> entries = {
+		std::make_pair("tenant0"_sr, TenantMapEntry(0, deterministicRandom()->coinflip())),
+		std::make_pair("tenant2"_sr, TenantMapEntry(2, deterministicRandom()->coinflip())),
+		std::make_pair("tenant3"_sr, TenantMapEntry(3, deterministicRandom()->coinflip())),
+		std::make_pair("tenant4"_sr, TenantMapEntry(4, deterministicRandom()->coinflip())),
+		std::make_pair("tenant6"_sr, TenantMapEntry(6, deterministicRandom()->coinflip()))
```

Comment:
I wouldn't do this coinflip here. This code is not exercising the encrypted property of the tenant map entry at all, and I think having the randomness there might be misleading. Rather, the encrypted property should be evaulated in the TenantManagementWorkload.

Reply:
Make sense; sorry I think I suggested Nim in my previous comment.